### ISSUE TITLE
Accessibility Help: Don't use monaco-editor, but semantic html

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26887]
The accessibility help menu uses a monaco-editor to present the help text:

![Code_-_Insiders_ZZmWt6dE4v](https://github-com/microsoft/vscode/assets/2931520/ff5416dd-646f-44b2-8e7d-5609f2254fd6)

I think we should try to use plain (semantic) html instead:

* I'm not so sure that the text editor actually improves accessibility (but I might be wrong).
   * The web offers many accessibility features and is optimized for rendering documents to read. I'd be very suprised if putting a document in a monaco-editor would give better accessibility features than using native web-features (e.g. `<p>`, `<h1>` elements)
   * NVDA has a special browse mode to navigate between headings (using `h`) and sentences (using arrow keys). If this mode does not work in VS Code, we should try to fix it.
   * I don't think character-level navigation is that useful for reading paragraphs.
   * [The NVDA help itself is a static html document that is not in a text editor](https://www.nvaccess.org/files/nvda/documentation/userGuide.html)
* It is inconsistent with other places where we render text (e.g. the trusted workspaces dialog)

FYI -isidorn
